drata/README.md

```diff
@@ -0,0 +1,42 @@
+# Drata
+
+## Overview
+
+This integration allows [Drata][5] customers to forward their DoControl-related logs and events to Datadog through automated security workflows.
+
+## Setup
+
+To set up this integration, you must have an active [Drata account][6]. You must also have proper [admin permissions][7] in Datadog.
+
+### Installation
+
+No installation is required on your host.
+
+
+#### Create an API key in Datadog
+
+1. Use Datadog's [Add an API key][2] documentation to create an API key. Give the key a meaningful name such as `Drata`.
+2. Copy the `Key` and save it.
+
+
+#### Create an application key in Datadog
+
+1. Use Datadog's [Add application keys][3] documentation to create an application key.
+2. Copy and save your application key.
+
+![Get_DD_Application_Key](https://raw.githubusercontent.com/DataDog/integrations-extras/master/docontrol/images/Get_DD_Application_Key.png)
```

Comment:
Currently, the highlighted text and options in this screenshot are too small to easily read. Could you re-take this screenshot so that they are more larger and more readable?

Reply:
New screenshot taken. the new one likely won't work until the image is on master.